indico/core/db/sqlalchemy/util/models.py

```diff
@@ -30,9 +30,39 @@
 from indico.core import signals
 
 
+class IndicoBaseQuery(BaseQuery):
+    def paginate(self, page=1, per_page=40, show_all=False):
```

Comment:
20 or 25? 40 are a bit much, you actually need to scroll on a big screen with 40 events